src/snapshot/tosvg.js

```diff
@@ -88,7 +88,7 @@ module.exports = function toSVG(gd, format, scale) {
     // fill whatever container it's displayed in regardless of plot size.
     svg.node().style.background = '';
 
-    svg.selectAll('text')
+    svg.selectAll('text,tspan')
```

Comment:
Please add test in `snapshot_test.js` similar to:

https://github.com/plotly/plotly.js/blob/795220f815afcf79c73885715379b606afc1b4a1/test/jasmine/tests/snapshot_test.js#L258-L307

but for `indicator` `<tspan>`s.

Reply:
Done in e6d8525ab07c56ca10bdbac6762b06d1d57bf830